AutomatedTesting/Gem/PythonTests/Atom/atom_utils/atom_constants.py

```diff
@@ -316,13 +316,58 @@ def depth_of_field(property: str = 'name') -> str:
             Use editor_entity_utils EditorEntity.add_components(list) to add this list of requirements.\n
           - 'Camera Entity' an EditorEntity.id reference to the Camera component required for this effect.
             Must be a different entity than the one which hosts Depth of Field component.\n
+          - 'CameraEntityId Override' Override enable for CameraEntityId, bool.
+          - 'Enabled Override' Override enable for Enabled, bool.
+          - 'QualityLevel Override' Override enable for QualityLevel, bool.
+          - 'ApertureF Override' Override enable for ApertureF, range 0.0-1.0.
+          - 'FocusDistance Override' Override enable for Focus Distance, range 0.0-1.0.
+          - 'EnableAutoFocus Override' Override enable for EnableAutoFocus, bool.
+          - 'AutoFocusScreenPosition Override' Override enable for AutoFocusScreenPosition, range 0.0-1.0.
+          - 'AutoFocusSensitivity Override' Override enable for AutoFocusSensitivity, range 0.0-1.0.
+          - 'AutoFocusSpeed Override' Override enable for AutoFocusSpeed, range 0.0-1.0.
+          - 'AutoFocusDelay Override' Override enable for AutoFocusDelay, range 0.0-1.0.
+          - 'EnableDebugColoring Override' Override enable for EnableDebugColoring, bool.
+          - 'Enable Depth of Field' Enables ot disables depth of field, bool.
+          - 'Quality Level' 0 or 1, 0 is standard Bokeh blur, 1 is high quality Bokeh blur.
+          - 'Aperture F' The higher the value the larger the aperture opening, range 0.0 to 1.0.
+          - 'F Number' The ratio of the system's focal length to the diameter of the aperture.
+          - 'Focus Distance' The distance from the camera to the focused object.
+          - 'Enable Auto Focus' Enables or disables auto focus.
+          - 'Focus Screen Position' XY value of the focus position on screen.
```

Comment:
is this the autofocus position? manual focus uses a distance not a position. 
specify that this is looking for math.Vector2(float x, float y) where these are 0.0 to 1.0 viewport dimensions.
"autofocus position (math.Vector2(float x, float y) where ranges are 0.0 to 1.0)"

Reply:
Fixed, mentioned it is related to autofocus even though that is not in the tooltip, and it is obvious if you ever launched the editor and looked at DoF